presenters.md

```diff
@@ -0,0 +1,136 @@
+---
+layout: page
+order: 0
+title: "Presenters: Confirm Participation"
+titleDisplay: "Presenter"
+---
+
+<p>In order to confirm your participation in Our Networks 2020, please complete the below form by Friday July 10, 2020.</p>
+
+<hr />
+
+<form action="https://formspree.io/orga@ournetworks.ca" class="form-container" method="POST">
```

Comment:
```suggestion
<form action="https://ournetworks.ca/presenters-confirm?next=TODO" class="form-container" method="POST">
```

This will use the form submission thingie I added. We should replace `TODO` with a URL for where to redirect after submitting.

Reply:
Reusing our query string callback https://github.com/ournetworks/ournetworks.ca/pull/184/commits/ff83283150dec17270ff4deff705b22df4864e7e#diff-2c38b3c8e67d8f313dd3a5b793cf11e9R12